packages/backend/server.js

```diff
@@ -33,6 +22,25 @@ const corsOptions = () => {
     credentials: true,
   };
 };
+
+const app = express();
+const httpServer = createServer(app);
+initializeSocketConnection(httpServer, corsOptions);
+
+// Database
+mongoose.set('strictQuery', false);
+mongoose
+  .connect(process.env.CONNECTION_STRING, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
```

Comment:
The options object on mongoose connect was removed because, since the last few versions, these options are default.